node/src/daemon/setup_reporter.rs

```diff
@@ -2437,6 +2517,40 @@ mod tests {
         assert_eq!(result.get("gas-price").unwrap().value, "10".to_string());
     }
 
+    #[test]
+    fn mapping_protocol_is_blanked_out() {
+        let _guard = EnvironmentGuard::new();
+        let data_directory =
+            ensure_node_home_directory_exists("setup_reporter", "mapping_protocol_is_blanked_out");
+        let conn = DbInitializerReal::default()
+            .initialize(&data_directory, DbInitializationConfig::test_default())
+            .unwrap();
+        let mut persist_config = PersistentConfigurationReal::from(conn);
+        persist_config
+            .set_mapping_protocol(Some(AutomapProtocol::Pcp))
+            .unwrap();
+
+        let setup = vec![
+            // blanked-out config-file setting
```

Comment:
I don't understand the purpose of this comment, do you? Might it be brought in from a copied test?

Reply:
All I can think of is that it may be trying to simulate a blanked-out mapping-protocol in a config file.

If so, it's a little silly. I'll take it out.